js/main.js

```diff
@@ -0,0 +1,91 @@
+var button = document.getElementById('buttonNews');
+button.onclick = function displayAndHide() {
+    var hide = document.getElementById('displayInfo');
+    if (hide.style.display === "") {
```

Comment:
The logic is clear and simple. But improve it to use a class name instead of style.display, so add a class called "hidden" for example, and define it in CSS. Then remove it when clicking again.

As an extra challenge, Google CSS transitions - you can use the CSS class to have an animation when hiding and displaying (that won't be possible if you .style.display)